dashboard/README.md

```diff
@@ -22,6 +22,20 @@ The learn.code.org code is segmented into three parts:
 
 See the main website-ci/README for server install and running instructions.
 
+## Code Style
```

Comment:
Hmm, good question. I feel like we're moving toward a world where any given developer is building one of:

* dashboard + pegasus (working from the repository root)
* dashboard + pegasus + code-studio
* dashboard + pegasus + apps
* dashboard + pegasus + code-studio + apps
* code-studio alone
* apps alone

There was a time where it was common to run dashboard in isolation, but I don't know if that's still true.